deploy/kiali/kiali_cr.yaml

```diff
@@ -96,7 +92,7 @@ spec:
 #
 # Allows for controlling what namespaces/projects are returned by Kiali.
 #
-# 'exclude' is optional takes an list of namespace/projects to be excluded from the list
+# 'exclude' is optional and takes a list of namespaces to be excluded from the list
```

Comment:
Does `exclude` support regex?

Reply:
I think. I dunno.  All I did was change this to be proper English.

Reply:
Yes, it says so in the following sentence. And we know it does because the default value uses regex.